src/Adapter/MSTestAdapter.PlatformServices/Utilities/DeploymentUtilityBase.cs

```diff
@@ -272,7 +272,7 @@ protected IEnumerable<string> Deploy(IList<DeploymentItem> deploymentItems, stri
     /// <param name="isDirectory">Is this a directory.</param>
     /// <returns>Paths to items to deploy.</returns>
     [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1031:DoNotCatchGeneralExceptionTypes", Justification = "Requirement is to handle all kinds of user exceptions and message appropriately.")]
-    protected string[]? GetFullPathToFilesCorrespondingToDeploymentItem(DeploymentItem deploymentItem, string testSource, string resultsDirectory, IList<string> warnings, out bool isDirectory)
+    protected List<string>? GetFullPathToFilesCorrespondingToDeploymentItem(DeploymentItem deploymentItem, string testSource, string resultsDirectory, IList<string> warnings, out bool isDirectory)
```

Comment:
IList?

Reply:
This is only internal/private stuff so I prefer to always be as specific as possible.